rancher/dashboard
# UI/update app source

### Summary
Fixes # https://github.com/epinio/ui/issues/171

Allows the user to edit an application and redeploy from another source

### How to test:
- Go to https://localhost:8005/epinio/c/default/applications
- Click on one of your application and check the Config tab (top right), the applications source tab **shouldn't** be displayed!
- Click on the Actions button (next to Config, 3 dots) and click Edit Config, now the Source tab should show with the wizard.
- Choose a different source/application and follow the steps, everything should stay the same (envs, configs,...) but the app source should be different.
- 
### Screenshot/Video
